benchmarks/perf-tool/sample-configs/filter-spec/filter-4-spec.json

```diff
@@ -33,12 +33,12 @@
                         {
                             "term":
                             {
-                                "color": "green"
+                                "taste": "green"
```

Comment:
is this change correct? green is still a color, shouldn't it be "sweet" like in other edits

Reply:
good catch. This is again a typo from my side.